src/pages/css/properties/background-image-property/index.md

```diff
@@ -1,15 +0,0 @@
----
-title: Background Image Property
----
-## Background Image Property
```

Comment:
For the sake of consistency, can you rename the other `Background Image` article to `Background Image Property`?